sql/src/main/java/io/crate/protocols/postgres/SimplePortal.java

```diff
@@ -112,7 +112,7 @@ public Portal bind(String statementName,
                 fields(),
                 resultReceiver, maxRows, this.params, sessionContext, portalContext);
             return portal.bind(statementName, query, statement, params, resultFormatCodes);
-        } else if (this.statement != null) {
+        } else if (this.statement != null && this.analysis != null) {
```

Comment:
Came across another problem here that the `BatchPortal` creation fails if the previous analysis could not be created due to a permission problem.